ui/apps/platform/src/messages/vulnMgmt.messages.ts

```diff
@@ -33,6 +33,10 @@ export const imageScanMessages = {
         header: 'Image out of scope for Red Hat Vulnerability Scanner Certification.',
         body: 'The scan results are not certified, as the base RHEL image is out of scope for certification. Please consider updating the base image.',
     },
+    contentSetUnavailable: {
+        header: 'Unavailable or invalid CPE data',
+        body: 'The NodeInventory is either missing CPE data or is holding invalid CPE data.',
```

Comment:
This is a customer-facing message. I don't think mentioning `NodeInventory` is meaningful for them. I suggest we choose a message such as  "The scan results are unavailable because CPE data was not found when scanning this node" or something similar.

Reply:
Shouldn't this be in `nodeScanMessages`?

Reply:
You're right, I moved it to `nodeScanMessages` and changed the wording